### DISCUSSION TITLE
!! Avoid use of upper cases in mqttany.yml !!

### DISCUSSION THREAD

**Discussion by EKNYESE** [CrazyIvan359/mqttany#156] [Troubleshooting]
Just to prevent spending too much time on debugging:
It seems, that **you cannot use UPPER CASES** in mqttany.yml - especially when defining the names/ device-ids of e.g. onewire-sensors.

Anyone who can confirm this anomaly?

**Comment by CrazyIvan359** [comment]
Upper case should not be a problem for device names, IDs, or OneWire IDs. Can you provide an example?
